integration-tests/tests/api/sign-up.spec.ts

```diff
@@ -22,6 +37,40 @@ test.concurrent('should auto-create an organization for freshly signed-up user',
   expect(result.organizations.total).toBe(1);
 });
 
+test.concurrent(
+  'freshly signed-up user should have a Hobby plan with 7 days of retention',
+  async () => {
+    const { ownerToken, createPersonalProject } = await initSeed().createOwner();
+    const result = await execute({
+      document: gql(/* GraphQL */ `
+        query organizations {
+          organizations {
+            total
+            nodes {
+              id
+              name
+            }
+          }
+        }
+      `),
+      authToken: ownerToken,
+    }).then(r => r.expectNoGraphQLErrors());
+
+    expect(result.organizations.total).toBe(1);
+
+    const { target } = await createPersonalProject(ProjectType.Single);
+
+    await waitFor(5_000); // wait for rate-limit to update (longer then LIMIT_CACHE_UPDATE_INTERVAL_MS)
```

Comment:
should we just import `LIMIT_CACHE_UPDATE_INTERVAL_MS + 1`?

Reply:
https://github.com/kamilkisiela/graphql-hive/pull/949/commits/bc235666e96617969d7e91dc269ad7636596d5f7